### ISSUE TITLE
Navigation instantly says "No route could be found" after destination is entered

### ISSUE THREAD

**Issue by lavglaab** [dyedgreen/pebble-directions#1]
The app doesn't seem to work at all anymore, which is a shame with how polished it is overall. I've tested it on a Pebble Time Round and Pebble Time Steel both on Rebble services.

Steps to reproduce:

1. Select a navigation mode from the 4 available
2. Dictate your destination
3. App will process for less than a second and say "No route could be found"

**Comment by dyedgreen** [comment]
Hi @piggehperson thanks for the feedback! I haven't used my Pebble in a while, and haven't done any maintenance for this project since then.

The app itself should still work, but from what you describe, the API keys probably expired. I'm not sure how to update apps on the Rebble store, but if you are happy to side-load, you should be able to keep using the app by cloning this repository and adding a file called `/src/js/apiKeys.js`.

The file should export:
```javascript
module.exports.hereAppId = YOUR_APP_ID; // API app id for here
module.exports.hereAppCode = YOUR_APP_CODE; // API app code for here
module.exports.mapQuestKey = YOUR_MAP_QUEST_KEY; // API key for map quest
```

When I originally made the app, both these services offered free tiers, which should more than suffice for personal usage 😃

Hope this helps, and thanks for still using the app (or rather, for trying to still use the app 😄)

**Comment by Threnklyn** [comment]
Hey, it looks like the routes for the api have changed:

* https://geocoder.ls.hereapi.com/6.2/geocode.json
* https://route.ls.hereapi.com/routing/7.2/calculateroute.json

**Comment by Threnklyn** [comment]
Oh and here's the link for updates or submissions to the rebble store:
https://github.com/pebble-dev/wiki/wiki/Preparing-a-new-app-for-the-Rebble-App-Store

**Comment by dyedgreen** [comment]
@Kuchenm0nster, thank you for submitting the PR, however I can't really verify if the changes made are correct/ will work as previously advertised. I also currently don't really have time to figure out how to build the project again (were to obtain the legacy Pebble dev kit etc.).

If you're up for it, I'd invite you to fork the repository and keep maintaining things there. I can then link to the new repository from here and would also be happy for you to submit the updated version to the REBBLE store. Let me know if that works for you 😃

**Comment by Threnklyn** [comment]
Sure
Since the only maintenance I did was changing the api urls I guess it won't be that much work.
JFYI rebble cloned the whole development webpage of pebble:
https://developer.rebble.io/developer.pebble.com/index.html
I will contact rebble regarding an update of the app

**Comment by Threnklyn** [comment]
Done
https://twitter.com/pebble_dev/status/1248597670819659778

If you want the homepage changed from your google form you have to send a short approval

**Comment by dyedgreen** [comment]
Yeah, I think it makes sense to point it to the repo or issue board for the repo. (Or wherever else people are able to get help/ send feedback).

Thanks for adopting the project! ☺️
